blockmanager.go

```diff
@@ -259,7 +259,21 @@ func (b *blockManager) Start() {
 	log.Trace("Starting block manager")
 	b.wg.Add(2)
 	go b.blockHandler()
-	go b.cfHandler()
+	go func() {
+		defer b.wg.Done()
+
+		log.Debug("Waiting for peer connection...")
+
+		// Before starting the cfHandler we want to make sure we are connected
```

Comment:
line looks long?

Reply:
79 characters, wrapping should start at 80 right?